header/core/listener.go

```diff
@@ -97,6 +101,7 @@ func (cl *Listener) listen(ctx context.Context, sub <-chan *types.Block) {
 
 			// broadcast new ExtendedHeader, but if core is still syncing, notify only local subscribers
 			err = cl.bcast.Broadcast(ctx, eh, pubsub.WithLocalPublication(syncing))
+			cl.lastHeaderBroadcast = time.Now().UnixMilli()
```

Comment:
Hmm, I don't really get why we would need this metric. It's also not part of the table.

Reply:
I can remove it if you want - source: you commented in #1040 that it wasnt in the table but would be useful to keep

Just respond here and i will either remove/resolve

Reply:
Quote from that issue:
 > The issue is about the last block served, not the header.
 
 Here we broadcast the header. Which IMO is an entirely useless metric.

Reply:
<img width="931" alt="image" src="https://user-images.githubusercontent.com/16523232/193302154-fbafca48-58ec-4612-8649-1b69802b97aa.png">

Reply:
will remove though, understood

Reply:
done